backend/services/implementations/schoolService.ts

```diff
@@ -52,6 +52,21 @@ class SchoolService implements ISchoolService {
     }
   }
 
+  /**
+   * This method gets all schools with the given country from the database.
+   */
+  async getSchoolByCountry(country: string): Promise<SchoolResponseDTO[]> {
```

Comment:
```suggestion
  async getSchoolsByCountry(country: string): Promise<SchoolResponseDTO[]> {
```
nit: can you rename the function to school**s**